### DISCUSSION TITLE
Toggle/OnOff button in Edit

### DISCUSSION THREAD

**Discussion by archgary** [cables-gl/cables_docs#467] [Feature Requests] [answered]
It might be cool to have an on/off button(OP) as a switch controller in the edit mode (rather than the screen mode). It could potentially turn into the threshold to open/close the stream in the patch. 

Like the shadow/darker part in this patch: https://cables.gl/edit/oc5-un
![Inkedchrome_jFvlHyC8Zw](https://user-images.githubusercontent.com/18259890/214068550-e8259bf2-684e-4bb6-ab09-07ea8f6f377c.jpg)

Or is there already an OP done the trick, please suggest!

Thank you for your time and results.

Best,
archgary

**Comment by pandrr** [comment, marked as answer]
there is 

https://cables.gl/op/Ops.Boolean.ToggleBool_v2

this is probably what you mean

**Reply by archgary** [reply]
Thanks! @pandrr , good hint. 

Use toggle bool to fix it as https://cables.gl/edit/oc5-un

<img width="1280" alt="chrome_q7zmm1AveZ" src="https://user-images.githubusercontent.com/18259890/214078518-ef7f79ec-7104-428f-8ddf-23a25d999099.png">

However, I would like to have it switched off, when you turn another switch on, I think there is no way to simplify that from here. That's the nature of this logic I think. 

Anyway, thanks for your help! :)
